Labyrinth/src/BoardDisplayer.java

```diff
@@ -56,10 +56,13 @@ public void paintCharacters(Graphics g){
 	 * @return
 	 */
 	public Point[] screenBounds(int x, int y){
+		//width is the number of tiles
+		//screenWidth is the number of pixels on the screen
+		//pixelsWide is number of pixels in a tile's width
 		int screenWidth = this.getWidth();
 		int screenHeight = this.getHeight();
-		int width = board.maxX() - board.minX();
-		int height = board.maxY() - board.minY();
+		int width = board.maxX() - board.minX() + 1;
+		int height = board.maxY() - board.minY() + 1;
```

Comment:
Why did this change?

Reply:
Width is the number of tiles. Subtracting the indices of the first and last tiles give you one less than the number of tiles.